experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/ExperimentVariantRepository.kt

```diff
@@ -48,6 +48,7 @@ class ExperimentVariantRepositoryImpl @Inject constructor(
                     key = it.variantKey,
                     weight = it.weight,
                     localeFilter = it.filters?.locale.orEmpty(),
+                    androidVersionFilter = it.filters?.androidVersion.orEmpty(),
```

Comment:
Ditto, if `locale` and `androidVersion` are not nullable we can ditch the `?.orEmtpy()`